datarepo-clienttests/src/main/java/scripts/testscripts/RetrieveSnapshot.java

```diff
@@ -34,14 +41,54 @@ public RetrieveSnapshot() {
 
   private SnapshotSummaryModel snapshotSummaryModel;
   private List<BlobId> scratchFiles = new ArrayList<>();
+  private boolean deleteProfile;
 
   public void setup(List<TestUserSpecification> testUsers) throws Exception {
-    // create the profile and dataset
-    super.setup(testUsers);
+    // Specify the name of a profile to use for the test with the TEST_RUNNER_BILLING_PROFILE_NAME
+    // env variable.
+    // If it's not specified or the profile isn't found, a new one with a randomly generated name
+    // will be created
+    String profileNameRaw = System.getenv("TEST_RUNNER_BILLING_PROFILE_NAME");
```

Comment:
Since this is using an env var, it could be initialized at class init time into a class constant.

Reply:
We also could grab the environmental variable in the TestConfiguration file, alongside the other environment variables. But not necessary. https://github.com/DataBiosphere/jade-data-repo/blob/develop/datarepo-clienttests/src/main/java/runner/config/TestConfiguration.java#L29